server/chat-plugins/scavenger-games.ts

```diff
@@ -361,6 +363,81 @@ const TWISTS: {[k: string]: Twist} = {
 			return true;
 		},
 	},
+
+	mistakes: {
+		id: 'mistakes',
+		name: 'Mistakes',
+		desc: 'After completing the hunt, players will guess what the most common incorrect answer for each question is.',
+		onAfterLoad() {
+			this.guesses = {};
+			this.incorrect = new Array(this.questions.length).fill('').map(() => ({}));
```

Comment:
```suggestion
			this.incorrect = new Array(this.questions.length).fill({});
```

Reply:
Oh I remember why I did this originally like this - .fill({}) filled it with references to the same object which doesn't work.  Gotta quickly PR and revert it back.